src/Raven.Client/Constants.cs

```diff
@@ -323,6 +323,13 @@ public class Folders
             }
         }
 
+        public static class Smuggler
```

Comment:
- internal
- non-static
- private ctor

I do not want one of them to be different that rest

Reply:
changed to `internal static`